src/types/index.d.ts

```diff
@@ -1,9 +1,9 @@
-import { Wallet, Signer } from "ethers";
 import { Provider } from "@ethersproject/abstract-provider";
 import { OpenAttestationDocument } from "@govtechsg/open-attestation";
 import { ErrorObject } from "ajv";
+import { Signer, Wallet } from "ethers";
 
-export type Network = "homestead" | "ropsten" | "rinkeby" | "local";
+export type Network = "homestead" | "ropsten" | "rinkeby" | "goerli" | "local" | "sepolia" | "matic" | "maticmum";
 type FormType = "TRANSFERABLE_RECORD" | "VERIFIABLE_DOCUMENT";
```

Comment:
network variables are being repeated in `constants/chainInfo.tsx`, any way to consolidate them?

Reply:
currently i cant leh, cos one is type and one is value.